runtime/altair/src/lib.rs

```diff
@@ -1140,6 +1198,24 @@ impl pallet_pool_system::Config for Runtime {
 	type WeightInfo = weights::pallet_pool_system::WeightInfo<Runtime>;
 }
 
+impl pallet_pool_registry::Config for Runtime {
+	type Balance = Balance;
+	type CurrencyId = CurrencyId;
+	type Event = Event;
+	type InterestRate = Rate;
+	type MaxSizeMetadata = MaxSizeMetadata;
+	type MaxTokenNameLength = MaxTrancheNameLengthBytes;
+	type MaxTokenSymbolLength = MaxTrancheSymbolLengthBytes;
+	type MaxTranches = MaxTranches;
+	type ModifyPool = pallet_pool_system::Pallet<Self>;
+	type Permission = Permissions;
+	type PoolCreateOrigin = EnsureSigned<AccountId>;
```

Comment:
We could use the type alias `PoolCreateOrigin`, we also use this in `PoolSystem`.

Reply:
That is a good catch! We should change this to democracy!

Reply:
The `PoolCreateOrigin` though, should be removed from `PoolSystem` ^^

Reply:
Can you explain why?

Reply:
We don't use it anywhere, as we create the pool from the registry.